Cargo.toml

```diff
@@ -28,7 +28,7 @@ aerospike-sync = {path = "./aerospike-sync", optional = true}
 aerospike-macro = {path = "./aerospike-macro", optional = true}
 
 [features]
-default = ["async", "serialization"]
+default = ["async", "serialization", "rt-tokio"]
```

Comment:
Seems better to have a default that can be switched off if required.... more of a personal choice though, might not want to sync this.